### ISSUE TITLE
Cannot work using Nixpkgs in docker container

### ISSUE THREAD

**Issue by skanehira** [skanehira/docui#124]
# Description
Using nixpkgs in docker contaienr, error occurred.

```
gorilla@5972c8e3de96:~$ docui
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x28 pc=0x87447b]

goroutine 1 [running]:
github.com/skanehira/docui/gui.(*info).display(0xc0003520c0)
        /build/source/gui/info.go:98 +0x3b
github.com/skanehira/docui/gui.newInfo(0xc000233820)
        /build/source/gui/info.go:92 +0x22e
github.com/skanehira/docui/gui.(*Gui).initPanels(0xc000233820)
        /build/source/gui/gui.go:157 +0xca
github.com/skanehira/docui/gui.(*Gui).Start(0xc000233820, 0xc000233820, 0xc00022eea0)
        /build/source/gui/gui.go:207 +0x2f
main.run(0xc0000ac120)
        /build/source/main.go:26 +0x221
main.main()
        /build/source/main.go:35 +0x74
```

**Comment by skanehira** [comment]
looks like when not mount `/var/run/docker.sock` this happen

**Comment by skanehira** [comment]
fix 2e78b17
